ndohyep/settings/dev.py

```diff
@@ -7,7 +7,7 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 
-try:
-    from .local import *
-except ImportError:
-    pass
+# try:
+#     from .local import *
+# except ImportError:
+#     pass
```

Comment:
Are you not finding this useful or is this because of pyflakes complaining? I added an ignore entry to the `flake8` block in the `setup.cfg` file, it now no longer complains about `import *` statements.